src/Search.php

```diff
@@ -8757,7 +8757,7 @@ public static function showItem($type, $value, &$num, $row, $extraparam = '')
                     $value = preg_replace('/' . self::LBBR . '/', '<br>', $value);
                     $value = preg_replace('/' . self::LBHR . '/', '<hr>', $value);
                     $value = '<div class="fup-popup">' . $value . '</div>';
-                    $valTip = "&nbsp;" . Html::showToolTip(
+                    $valTip = '' . Html::showToolTip(
```

Comment:
In most of otehr usages in the same class, `showTooltips()` results are separated with a simple space.
```suggestion
                    $valTip = ' ' . Html::showToolTip(
```

Reply:
(also, this is not a really result data and should not be present in exports or API results I guess)